.github/workflows/helmfile_staging_plan.yaml

```diff
@@ -35,13 +35,19 @@ jobs:
       - name: Load EnvVars
         run: |
             ./helmfile/getContext.sh true
-      - name: Run helmfile
-        id: helmfile_diff
-        uses: helmfile/helmfile-action@v1.0.0
+      - name: Setup helmfile
+        uses: mamezou-tech/setup-helmfile@v2.0.0
         with:
-          helmfile-args: diff --environment staging
+          install-kubectl: no
+          install-helm: yes
+      - name: Helmfile Diff
+        id: helmfile_diff
+        run: |
+          echo 'var<<EOF' >> $GITHUB_OUTPUT
+          helmfile --environment staging diff >> $GITHUB_OUTPUT
+          echo 'EOF' >> $GITHUB_OUTPUT
       - name: Helmfile Diff Comment
         uses: mshick/add-pr-comment@v2
         with:
           message: | 
-            Placeholder for helm diff once the output from GA works.
+            ${{join(steps.helmfile_diff.outputs.*, '\n')}}
```

Comment:
Oh that is the working diff here I assume?

Reply:
You got it!